### PR TITLE
Fixes part of #2003: Modified the use of space_8

### PR DESCRIPTION
**Pull request by Arjupta**
Fixes part of #2003, This commit removes the use of dimension space_8
with the dimensions of use-case specific names.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by aggarwalpulkit596 — APPROVED** [review]
LGTM

File: app/src/main/res/layout-land/submitted_answer_item.xml [hunk 1]
```diff
@@ -53,9 +53,9 @@
       android:layout_height="wrap_content"
       android:background="@drawable/submitted_answer_background"
       android:paddingStart="12dp"
-      android:paddingTop="@{viewModel.hasConversationView ? @dimen/space_16dp : @dimen/space_8dp}"
+      android:paddingTop="@{viewModel.hasConversationView ? @dimen/space_16dp : @dimen/submitted_answer_text_non_conversation_view_padding_top}"
       android:paddingEnd="12dp"
-      android:paddingBottom="@{viewModel.hasConversationView ? @dimen/space_16dp : @dimen/space_8dp}"
+      android:paddingBottom="@{viewModel.hasConversationView ? @dimen/space_16dp : @dimen/submitted_answer_text_non_conversation_view_padding_bottom}"
```
Comment (rt4914) [inline app/src/main/res/layout-land/submitted_answer_item.xml:58]: @ARJUPTA Really nice to this naming structure.

**Review by rt4914 — APPROVED** [review]
Really nice work @ARJUPTA 

Keep it up. Thanks.

**Comment by rt4914** [comment]
@aggarwalpulkit596 Suggest **not** giving points to this PR as the final PR will receive 150 points.

**Pull request merged**